lib/ansible/modules/network/meraki/meraki_network.py

```diff
@@ -265,8 +291,20 @@ def main():
             payload['tags'] = construct_tags(meraki.params['tags'])
         if meraki.params['timezone']:
             payload['timeZone'] = meraki.params['timezone']
-        if meraki.params['disable_my_meraki'] is not None:
+        if meraki.params['enable_my_meraki'] is not None:
+            if meraki.params['enable_my_meraki'] is True:
+                payload['disableMyMerakiCom'] = False
+            else:
+                payload['disableMyMerakiCom'] = True
+        elif meraki.params['disable_my_meraki'] is not None:
+            meraki.module.deprecate("meraki_network will stop supporting disable_my_meraki in Ansible 2.13. Please update your playbook.", version=2.13)
```

Comment:
* Name of the new option could be mentioned in the deprecation message.
* `2.13` doesn't need to be mentioned in deprecation message: `This feature will be removed in version 2.13. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.` will automatically be added as suffix.

Reply:
Updated.